### PR TITLE
Productlisting page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
create product listing page
Add filters
Add products in grid of 4
All filter functionalities are working

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+:root{
+  --dark-color: #0D0D0E;
+  --dark-brown: #5a5f57;
+  --text-white-color: #f3f2f1
 
+}
 .App {
 display: flex;
 flex-direction: column;
+/* background-color: var(--text-white-color) */
```
Comment (dusariudaykumar) [inline src/App.css:11]: You can remove the comments

**Review by therajatg — APPROVED** [review]
code looks clean

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 import "./Navbar.css"
+import {FiShoppingCart} from "react-icons/fi"
+import {BsHeart} from "react-icons/bs"
+import {AiOutlineUser} from "react-icons/ai"
 const  Navbar = () => {
     return (
         <>
                <div class="navbar col-12">
                <div class="nav-logo-box ">
                <img src = ""/>
```
Comment (dusariudaykumar) [inline src/components/Navbar/Navbar.jsx:11]: remove unused code

**Review by dusariudaykumar — APPROVED** [review]
the folder structure is good. 
 Just one suggestion remove the commented code then push it to GitHub

**Pull request merged**